test/src/runner/testRunnerSessionCreate.js

```diff
@@ -75,6 +73,11 @@ describe('testRunnerSessionCreate', function() {
     };
 
     return runTests({'reuse-browser': true, source: testsPath}, settings({
+      desiredCapabilities: {
+        'nightwatch:options': {
```

Comment:
should we call it `nw:options`, @beatfactor  @AutomatedTester thoughts?

Reply:
I don't have an opinion here either way. `nw` is short and easy for folks where `nightwatch` removes any ambiguity.